data/data/vsphere/variables-vsphere.tf

```diff
@@ -76,4 +76,6 @@ variable "vsphere_control_plane_num_cpus" {
 variable "vsphere_control_plane_cores_per_socket" {
   type = number
 }
-
+variable "disk_type" {
```

Comment:
@ayesha54 you will need the default string here since `diskType` is set to required. Or set `diskType` to `Required: false`